reader/components/highlighter/components/ContextMenu.jsx

```diff
@@ -17,8 +16,11 @@ class ContextMenu extends React.PureComponent {
       dispatch,
     } = this.props
     const { rects, selectedText, annotationId } = this.props
-    let toggleSidebar = noop
-    if (!ui.isEnhancementSidebarVisible && isEmpty(document.annotations))
+    let toggleSidebar = () => {}
+    if (
+      !ui.isEnhancementViewVisible &&
+      Object.entries(document.annotations).length === 0
```

Comment:
You use `entries` but technically `keys` might be better for the performance. It's not a big deal for an 5-keys object anyway.

Reply:
https://github.com/oacore/reader/pull/104/commits/d4b1c329fc788693ce580fa9b7ec9cfe759d33a6